demo/package.json

```diff
@@ -31,6 +31,9 @@
   },
   "jest": {
     "globals": {
+      "ts-jest": {
+        "tsconfig": "node_modules/@labkey/build/webpack/tsconfig.json"
```

Comment:
any reason that the demo module tsconfig.json file wasn't removed in this module?